README.md

```diff
@@ -235,6 +252,57 @@ like this:
 You'll probably get a more reliable experience if you use
 [forever](https://npmjs.org/package/forever) to keep the daemon running.
 
+### Optional Arguments
```

Comment:
This will be really useful, but it's probably better to leave it out for now, if it's just going to be empty.